Assets/MixedRealityToolkit/_Core/Utilities/UX/HoverLight.cs

```diff
@@ -51,28 +56,31 @@ public Color Color
 
         private void OnEnable()
         {
-            Initialize();
+            AddHoverLight(this);
         }
 
         private void OnDisable()
         {
-            UpdateHoverLight();
+            RemoveHoverLight(this);
+            UpdateHoverLights(true);
         }
 
+#if UNITY_EDITOR
         private void Update()
         {
-            if (Application.isPlaying || !Application.isEditor)
```

Comment:
Why did this get reverted?

Reply:
Personally, considering this is ONLY required in the editor, a compiler directive is better than an if, ensures it is not run and no performance check when running in prod